components/typography/index.tsx

```diff
@@ -18,17 +18,19 @@ type TypographyProps = {
 		| 'largeTitle'
 	weight: 'regular' | 'medium' | 'semiBold' | 'bold' | 'extraBold'
 	color: 'black' | 'gray' | 'white' | 'green'
+	colorImmutable?: boolean
 	tag?: string
 	muted?: boolean
 } & React.DetailedHTMLProps<React.HTMLAttributes<HTMLParagraphElement>, HTMLParagraphElement>
 
-const Typography = ({ children, variant, weight, color, tag, muted, ...props }: TypographyProps) => {
+const Typography = ({ children, variant, weight, color, colorImmutable, tag, muted, ...props }: TypographyProps) => {
 	const className = classNames(
 		styles.root,
 		styles[variant],
 		styles[color],
 		styles[weight],
 		{ [styles.muted]: muted },
+		{ [styles['color-immutable']]: colorImmutable },
```

Comment:
Please follow the naming convention above.
from:
`{ [styles['color-immutable']]: colorImmutable },`
to:
`{ [styles.colorImmutable]: colorImmutable },`

Reply:
Fixed. :+1: 
The reason why I've used a different naming convention was from seeing a few other classes using the kebab case convention.

Reply:
The CSS modules classes are kebab-case and then are used as camelCase on the JSX syntax.